tutorials/on-beyond-magpie0.md

```diff
@@ -0,0 +1,111 @@
+than one cent per text record (1,000 Unicode characters) and the first 5000 requests are free, programs that use
+it must be authenticated. Read [the instructions for creating an API key][auth].  A single key can be used for your entire class, but be careful to delete it after the class has used it to ensure they don't use it for other purposes.
+
+
+### Action
+
+Generate an API key for your project.
+
+## Accessing the API Interactively 
+
+If you are not familiar with the Cloud Natural Language API, first examine its abilities using the interactive feature in the ["Try the API"][nlp] section of the overview.  Notice it can extract entities, detect sentiment, and parse syntax.
+
+### Entity Detection
+
+While entity detection may seem to be a simple task, there's actually some significant processing going on.  Try using the sentence "The Eiffel Tower is one of many highlights in Paris, France." and examine the entities it extracts.  These should include "Paris" and "France."  While this may seem obvious, change the sentence to "The Eiffel Tower is one of many highlights in Paris, Texas."  The API no longer finds the entity "Paris" (the city in France).  Instead, it just finds the city "Paris, Texas."
+
+### Syntax Analysis
+Syntax analysis can handle some very complicated sentences.  Try something like the first sentence of [Alice in Wonderland][alice]:
+>Alice was beginning to get very tired of sitting by her sister on the bank, and of having nothing to do: once or twice she had peeped into the book her sister was reading, but it had no pictures or conversations in it, ‘and what is the use of a book,’ thought Alice ‘without pictures or conversations?’
+
+Not only does it recognize that "was" is a verb, it returns the base verb "be" and the mood, number, person, and tense of the verb.
+
+### Sentiment Analysis
+Sentiment analysis returns a score from -1.0 (for negative) to 1.0 (for positive) sentiment.  The magnitude is the strength of the sentiment.  If the text consists of more than one sentence, there will be a score for the entire text and each individual sentence.  Encourage students to try to create sentences with high magnitude. 
+
+## Accessing the API with the API Explorer
+
+The interactive feature of the WWW page does not show the JSON value returned by the API.  To see the returned JSON, use the [API Explorer][explorer].  You can select to analyze any one item (entities, sentiment, or syntax) or all three ([annotateText][annotate]).  (The pages describing many of the other APIs also have a "TRY IT!" button that will access the API Explorer directly.)
+
+This provides a way to introduce students to HTTP calls.  The details of the call parameters can be found [on-line][annotateapi].  In the API explorer, the context menu will show the properties that can be used.  As a minimum, in the `Request` body:
+
+* add a `document` property.  Within the `document` property 
```

Comment:
We capitalize these and treat them as sentences. I'll fix these, just a note for future reference.